mimic/model/keypair_objects.py

```diff
@@ -0,0 +1,30 @@
+"""
+Keypair objects for mimic
+"""
+
+from characteristic import attributes
+
+@attributes(['name', 'public_key', 'user_id'])
+class KeyPair(object):
+    """
+    A KeyPair object
+    """
+    static_defaults = {
+        "fingerprint": "35:9d:d0:c3:4a:80:d3:d8:86:f1:ca:f7:df:c4:f9:d8",
+        "private_key": """-----BEGIN RSA PRIVATE KEY-----\nMIICXAIBAAKBgQC9mC3WZN9UGLxgPBpP7H5jZMc6pKwOoSgre8yun6REFktn/Kz7\n
```

Comment:
Could we possibly [shell out]() to `ssh-keygen` for this instead?  Even "do not use" keys like this are a potential security nightmare for naive users.  Not to mention the fact that Github scanners looking for `BEGIN RSA PRIVATE KEY` will pick it up and we will get spurious bug reports opened :).